test/test_quantized_tensor.py

```diff
@@ -332,5 +333,18 @@ def test_qscheme_pickle(self):
 
         self.assertEqual(f2.qscheme, torch.per_tensor_symmetric)
 
+    def test_copy(self):
+        # deep copy
+        scale, zero_point, dtype = 1.0, 2, torch.uint8
+        q_int = torch.randint(0, 100, [3, 5], dtype=dtype)
+        q = torch._make_per_tensor_quantized_tensor(q_int, scale=scale, zero_point=zero_point)
+        qc = deepcopy(q)
+        self.assertEqual(qc, q)
+        # copy_
+        qd = torch._empty_affine_quantized([3, 5], scale=scale, zero_point=zero_point, dtype=torch.quint8)
```

Comment:
what if the scale, zero_point, dtype, type of quantization, etc are different?

Reply:
code is here: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Copy.cpp#L118-L123
it requires same quantization scheme and dtype, but allows to copy from different scale/zero_point.

Reply:
these are already tested in `test_qtensor_copy` actually, I'll update the test